src/central-server/admin-service/application/src/main/resources/application.yml

```diff
@@ -5,7 +5,7 @@ spring:
   jpa:
     open-in-view: false
     hibernate:
-      ddl-auto: validate
+      ddl-auto: none
```

Comment:
was this intentional?

Reply:
Intentional indeed, it was either this or provide users liquibase to CS as well